src/tchap/util/TchapRoomUtils.ts

```diff
@@ -38,15 +37,19 @@ export default class TchapRoomUtils {
      * @returns string that matches of one TchapRoomAccessRule //todo or null? or empty?
      */
     static getTchapRoomAccessRule(room: Room): TchapRoomAccessRule {
-        return room.currentState.getStateEvents(TchapRoomAccessRulesEventId, "")?.getContent().rule;
+        // the state event TchapRoomAccessRulesEventId is working only when the rule access is invite -> maybe a change in the backend configuration ? needs to check
+        // That's is why sometimes the last state events is undefined, here we are not directly using getJoinRule because of types compatibility
+        const lastEvent =  room.getLiveTimeline().getState(EventTimeline.FORWARDS)?.getStateEvents(EventType.RoomJoinRules)[0];
```

Comment:
RoomJoinRules is "m.room.join_rules" (standard matrix)
TchapRoomAccessRules is "im.vector.room.access_rules" (tchap specific)
Two different things ! See doc : https://www.notion.so/mercurial-timer-ec4/Sp-cificit-s-Tchap-vs-Element-d-tails-329cf4d20ce840a3b53e29dd2d76f0f0#84582964ab474317a2dc0e36f27c66bf